dist/templates/ovs-node.yaml.j2

```diff
@@ -86,8 +86,8 @@ spec:
             cpu: 100m
             memory: 300Mi
           limits:
-            cpu: 200m
-            memory: 400Mi
+            cpu: 500m
+            memory: 500Mi
```

Comment:
Wont we have the same issue again with 256 cores? Why not just remove the memory limit?

```
524288÷2048=256
```

Reply:
Yes, the issue would reoccur if the node have 256 cores. is it okay to get rid of the limit completely?

Reply:
*I* think its fine for upstream but Ill let the maintainers decide. Wait until they weigh in.

Reply:
256 cores isnt that far away anymore :)

Reply:
@girishmg wdyt?

Reply:
@arghosh93 did you check if we need to adjust anything downstream within OCP?

Reply:
@martinkennelly We run ovs-vswitchd as a systemd unit and add dropins[1] along with systemd unit[2] which comes with Openvswitch on RHEL. LimitSTACK is set in default ovs-vswitchd systemd unit[2].

[1] - https://github.com/openshift/machine-config-operator/blob/c74052cba27ebeb11a697d579ce99c4f47e47a0f/templates/common/_base/units/ovs-vswitchd.service.yaml
[2] - https://github.com/openvswitch/ovs/blob/master/rhel/usr_lib_systemd_system_ovs-vswitchd.service.in#L18

Reply:
Sweet! Thanks @arghosh93 
```
LimitSTACK= | ulimit -s | Bytes
```
https://www.freedesktop.org/software/systemd/man/systemd.exec.html

Reply:
haven't heard back from @girishmg yet, and this PR is not removing the limits. I think we can handle removing the limits in a follow up PR, and merge what we have here.